compiler/rustc_codegen_llvm/src/context.rs

```diff
@@ -675,6 +687,44 @@ impl<'ll, 'tcx> CodegenCx<'ll, 'tcx> {
         llvm::set_linkage(g, llvm::Linkage::AppendingLinkage);
         llvm::set_section(g, c"llvm.metadata");
     }
+
+    pub(crate) fn add_objc_module_flags(&self) {
```

Comment:
We aren't adding `"Objective-C Version"`, I assume that is intentional?

Reply:
It was not intentional. Added.